### PR TITLE
Parameterize gitea config options

### PR DESCRIPTION
**Pull request by lukrop**
Parameterize the following Gitea config options:

`DISABLE_REGISTRATION`, `REQUIRE_SIGNIN` and `ENABLE_OPENID_SIGNIN`

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -163,3 +163,7 @@ config:
 #  secretKey: define_your_secret
   disableInstaller: false
   offlineMode: false
+  require_signin: false
```
Comment (gs11) [inline values.yaml]: Please use lowerCamelCase for all parameters

**Review by gs11 — COMMENTED** [review]
Thanks for the PR! Just a comment on the parameter naming

File: values.yaml (hunk 1 above)
Comment (jfelten) [reply values.yaml]: Thank you for the PR. - LGTM, but please address @gs11 's comments.  I will merge it afterwards.

**Review by jfelten — CHANGES_REQUESTED** [review]
Please use camel case for all variables.  Once done I will approve and merge

File: values.yaml (hunk 1 above)
Comment (lukrop) [reply values.yaml]: Done!

**Review by gs11 — APPROVED** [review]

**Review by jfelten — APPROVED** [review]

**Pull request merged**